conduct.md

```diff
@@ -132,7 +132,13 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the class conduct guide as specified by the unacceptable conduct sections. It is important that the conduct guide is upheld and acceptable conduct is followed in order to support a positive learning environment in the class and online.
+
+When a violation is committed, the student is given a penalty for the total amount of violations committed during the semester.  The only exception to this is that Honor Code violations, and the penalties associated with them, are handled and investigated by the Honor Committee.
+
+If you believe that a violation was given incorrectly or unfairly, it is possible to meet with the teaching assistants and the professor of the class to discuss with them a possible appeal for the violation.
```

Comment:
Instead of "a violation was given", it might make more sense to say "a violation was reported".

Reply:
It seemed that in issue #8, we were converging on deciding to involve only the professor in appeals. Is this still the case? If so, then "the teaching assistants" should be removed. Additionally, you may want to add at the end that the professor makes the final decision.

Reply:
Ah, I must of missed that part in the issue. I removed them in the new commit.